content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Synthetic Test Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. You are notified when the Synthetic test monitor alerts.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
```

Comment:
Can we change this image to this one (so Synthetics -> Monitors versus Monitors -> Synthetics): https://share.getcloudapp.com/yAuKOoKW (let me know if we need a new screenshot)